gatsby-browser.js

```diff
@@ -145,12 +146,14 @@ export const onClientEntry = () => {
                 j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
                 'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
                 })(window,document,'script','dataLayer','GTM-TNX2ZKH');`,
```

Comment:
Same here.. Looks like this GTM id is outdated. Please speak with BI team about this dataLayer, which is not firing